gameMechanics.js

```diff
@@ -48,15 +48,21 @@ var insertDisc = function(currentGrid, selectedColumn, currentPlayer) {
 			currentGrid[selectedColumn][position] = playerTwo;
 		}
 	}
+	turn = alternateTurns(currentPlayer);
 	return currentGrid;
 }
 
-var stackCheck = function(input) {
+var connectCheck = function(input) {
 // stub function
 }
 
-var connectCheck = function(input) {
-// stub function
+var alternateTurns = function(currentTurn) {
+	var playerOne = 0;
+	var playerTwo = 1;
```

Comment:
I suggest using a Constant or an Enumeration to represent these values; it'll be less confusing to figure out whose turn it is.